website/settings/defaults.py

```diff
@@ -74,6 +74,21 @@ def parent_dir(path):
 MAIL_USERNAME = 'osf-smtp'
 MAIL_PASSWORD = ''  # Set this in local.py
 
+# Notification options
+SUBSCRIPTIONS_AVAILABLE = {
```

Comment:
These constants aren't meant to configurable, so they don't really belong in `website.settings`. Maybe move them to `website.notifications.constants`.